services/bettersocial/chat/updateBetterChannelMembers.js

```diff
@@ -58,7 +59,7 @@ const updateBetterChannelMembers = async (
 
   if (withUpdate) {
     try {
-      await chatChannel.updatePartial({
+      channel = await chatChannel.updatePartial({
         set: defaultUpdateData
       });
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `await chatChannel.updatePartial({ set: defaultUpdateData });` to `channel = await chatChannel.updatePartial({ set: defaultUpdateData });` is a good one. It ensures that the updated channel state is captured and can be returned by the function. This will help in maintaining the latest state of the channel throughout the application.



<!-- This is an auto-generated comment by OSS CodeRabbit -->